README.md

```diff
@@ -12,90 +12,110 @@
   <img alt="GitHub closed issues" src="https://img.shields.io/github/issues-closed/posthog/posthog"/>
 </p>
 
-## PostHog is an open-source product analytics suite, built for engineers
-* Automatically track every event on your website or app
-* Understand your users and how to improve your product
-* Deploy on your own infrastructure to keep control of your data.
+<p align="center">
+  <a href="https://posthog.com/docs">Docs</a> - <a href="https://posthog.com/using-posthog">Using PostHog</a> - <a href="https://posthog.com/slack">Support community</a> - <a href="https://posthog.com/roadmap">Roadmap</a> - <a href="https://github.com/PostHog/posthog/issues/new?assignees=&labels=bug&template=bug_report.md">Bug report</a>
+</p>
+
+## PostHog is an open-source suite of product and data tools, built for engineers
+
+- Automatic and/or customized event capture for your app and website
```

Comment:
May be worth splitting out into two points?

```suggestion
- Autocapture events, so you spend less time on instrumentation
- Track events manually, so you get the detail you need
```

Alternatively, I like the fact that all other points in this list start with a verb, so making consistent with that:

```suggestion
- Specify events manually, or use autocapture to get started quickly
```